src/sql_engine/src/query/mod.rs

```diff
@@ -15,12 +15,44 @@
 ///! Module for representing how a query will be executed and values represented
 ///! during runtime.
 pub mod plan;
-pub mod transform;
 
-use crate::query::plan::PlanError;
+mod transform;
+
+pub use plan::{Plan, PlanError, SchemaCreationInfo, TableCreationInfo};
+pub use transform::QueryProcessor;
```

Comment:
Could you please not to reexport, unless it is going to be a public API of cargo module